homeassistant/components/sonos/switch.py

```diff
@@ -0,0 +1,224 @@
+"""Entity representing a Sonos battery level."""
```

Comment:
Fix file docstring: Should be "Sonos switch"

Reply:
Thanks! I will change it!